docker-files-for-Gitlab-CI-rust/substrate-ci/Dockerfile

```diff
@@ -34,9 +40,7 @@ RUN set -eux; \
 # compiler ENV
 ENV CC=gcc \
 	CXX=g++ \
-# cache handlers
+# cache handler
 	RUSTC_WRAPPER=sccache \
-	CARGO_HOME=/ci-cache/substrate/cargo \
-	SCCACHE_DIR=/ci-cache/substrate/sccache \
```

Comment:
why is this not needed anymore?

Reply:
deleting these lines and mentioning them in the specific projects will make this image universal.